packages/mask/src/plugins/Collectible/SNSAdaptor/Tabs/OffersTab.tsx

```diff
@@ -0,0 +1,63 @@
+import { LoadingBase, makeStyles } from '@masknet/theme'
+import { useMemo } from 'react'
+import { CollectibleTab } from '../CollectibleTab'
+import { EMPTY_LIST } from '@masknet/shared-base'
+import { Typography, Button } from '@mui/material'
+import { Icons } from '@masknet/icons'
+import { CollectibleState } from '../../hooks/useCollectibleState'
+import { NFTOfferCard } from '../../../../components/shared/NFTCard/NFTOfferCard'
+import { useI18N } from '../../../../utils'
+
+const useStyles = makeStyles()((theme) => ({
+    body: {
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        minHeight: 300,
+        gap: 12,
+        justifyContent: 'center',
+    },
+    emptyIcon: {
+        width: 36,
+        height: 36,
+    },
+    emptyText: {
+        fontSize: 14,
+        color: theme.palette.maskColor.second,
+    },
+}))
+
+export function OffersTab() {
+    const { classes } = useStyles()
+    const { orders, provider } = CollectibleState.useContainer()
+    const _orders = orders.value?.data ?? EMPTY_LIST
+    const { t } = useI18N()
+    return useMemo(() => {
```

Comment:
No need to use `useMemo` here, you can wrap with `memo`

```tsx
export const OffersTab = memo(function OffersTab() {
 //
})
```

Reply:
ok

Reply:
Don't forget to edit other tabs as well.

<img width="292" alt="image" src="https://user-images.githubusercontent.com/1141198/184300225-6a658879-0272-4ae0-bf11-0de157b29d4b.png">